config/doctrine/InvoiceSequence.orm.xml

```diff
@@ -11,6 +11,9 @@
         </id>
         <field name="index" column="idx" type="integer" />
         <field name="version" type="integer" version="true" />
+        <field name="year" type="integer" nullable="true"/>
+        <field name="month" type="integer"/>
+
```

Comment:
Why not having a sequence_type and an enum of possibilities? If we want to improve this or change the behavior we would be obliged to add one or more columns.
WDYT?